infra/index.ts

```diff
@@ -39,6 +39,9 @@ const debeziumTopic = new gcp.pubsub.Topic('debezium-topic', {
   'alerts-updated',
   'settings-updated',
   'update-comments',
+  'post-scout-matched',
+  'submission-changed',
+  'community-link-submitted',
```

Comment:
There's some misalignment with the names here. They don't follow the same convention